tests/e2e/cucumber/steps/ui/adminSettings.ts

```diff
@@ -242,3 +242,35 @@ When(
     }
   }
 )
+
+When(
+  /^"([^"]*)" changes (userName|displayName|email|password|role) to "([^"]*)" for user "([^"]*)" using the sidebar panel$/,
+  async function (
+    this: World,
+    stepUser: string,
+    attribute: string,
+    value: string,
+    user: string
+  ): Promise<void> {
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const usersObject = new objects.applicationAdminSettings.Users({ page })
+
+    // deleting a user with a same userName before testing
+    if (attribute === 'userName') {
+      const newUser = this.usersEnvironment.createUser({
+        key: value,
+        user: {
+          id: value,
+          displayName: '',
+          password: 'password',
+          email: ''
+        }
+      })
```

Comment:
Out of curiosity: why not call `this.usersEnvironment.getUser()` here instead?

Reply:
then we get error `user with key anna not found`. We need create `anna` in the store before deleting
https://github.com/owncloud/web/blob/master/tests/e2e/support/environment/userManagement.ts#L8-L9